builder/lxc/lxc_config.go

```diff
@@ -0,0 +1,43 @@
+package lxc
+
+import (
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+type LxcConfig struct {
```

Comment:
I think if you make LxcConfig private then you enforce that people have to use the `NewLxcConfig` constructor http://www.golangpatterns.info/object-oriented/constructors

It's a small thing but worth mentioning